c7n/mu.py

```diff
@@ -624,7 +623,7 @@ class PolicyLambda(AbstractLambdaFunction):
     """Wraps a custodian policy to turn it into lambda function.
     """
     handler = "custodian_policy.run"
-    runtime = "python%d.%d" % sys.version_info[:2]
```

Comment:
ugh.. no not yet. we're not even done with 3.6 compatibility. we're hyper focused on stability and compatibility (forward in this context), not pushing mass migrations.